ddtrace/profiling/profiler.py

```diff
@@ -126,7 +126,7 @@ class _ProfilerInstance(service.Service):
         factory=lambda: formats.asbool(os.environ.get("DD_PROFILING_MEMORY_ENABLED", "True")), type=bool
     )
     enable_code_provenance = attr.ib(
-        factory=attr_utils.from_env("DD_PROFILING_ENABLE_CODE_PROVENANCE", True, formats.asbool),
+        factory=attr_utils.from_env("DD_PROFILING_CODEHOTSPOTS_ENABLED", True, formats.asbool),
```

Comment:
we need to still support `DD_PROFILING_ENABLE_CODE_PROVENANCE`?

Reply:
We don't need to based on our version policy but we probably should out of courtesy and since it's easy to do.

Bare minimum we should mention DD_PROFILING_ENABLE_CODE_PROVENANCE in the release note for the migration, which is done 👍.